templates/questions/index.html.ep

```diff
@@ -291,7 +291,10 @@ $(document).ready(function() {
 			Privacy Policy
 		</div>
 		<div class="privacyContent">
-			Here is the content of the privacy policy. It is kinda long, so we gotta make sure it wraps. Hopefully people really like this privacy policy. Hopefully it does not scare people away.
+			AMA uses "cookies" to enhance User experience. User's web browser places cookies on their hard drive for record-keeping purposes. Users are still completely anonymous. User may choose to set their web browser to refuse cookies, or to alert when cookies are being sent. If cookeis are disabled, note that some parts of the AMA site may not function properly.
+			<br><br>AMA also gives the user the option to sign in. When the user is signed in, the User is still entirely anonymous to all other users. However, when signed, it is possible for site moderators to track down specific people thorugh the database if neccesarry. Notice that if a user signs in, he/she loses an element of anonymity.
```

Comment:
Per our excellent debate the other day, I thought users do *not* sign in to AMA?  But it seems your description here claims they do?

> it is possible for site moderators to track down specific people thorugh the database if neccesarry

thorugh -> through
neccesarry -> necessary

Why would it ever be "*necessary*"??